Gemfile

```diff
@@ -13,7 +13,7 @@ gem "kaminari", ">= 1.2.0"
 gem "pundit"
 
 # Adds health check functionality
-gem "health_check", ">= 3.0.0"
+gem "health_check", github: "/ianheggie/health_check", ref: "0b799ead604f900ed50685e9b2d469cd2befba5b"
```

Comment:
Wait, so where did the file with the comments come from, if not from the 'official' release?

Reply:
This was utterly irritating - the option to not include the sensitive information was not in 3.0.0 but in edge release. So the readme file on that repo is inaccurate - if you use 3.0.0 version you will not get the configuration to disable sensitive info. Had to compare the two files side by side